meson.build

```diff
@@ -212,10 +212,8 @@ if not get_option('snpe-support').disabled()
       SNPE_ROOT = cmd.stdout().strip()
       if SNPE_ROOT != ''
         message('Got SNPE_ROOT: @0@'.format(SNPE_ROOT))
-        # TODO: This is not a portable way to check if a directory exists
-        # After bumping up to 0.53.0, the following line can be replaced with one that uses FS module
-        snpe_dir_not_exist = run_command('[', '-d', SNPE_ROOT, ']', check : false).returncode()
-        if snpe_dir_not_exist != 0
+        fs = import('fs')
```

Comment:
suggestion: declare `fs = import('fs')` in root meson.build and use it in the project.

Reply:
Thank you. I updated.